### PR TITLE
Add ShapeCompletion Action

### PR DESCRIPTION
**Pull request by chancecardona**
Resolves [7586](https://github.com/PickNikRobotics/moveit_studio/issues/7586)

### CODE REVIEW

File: moveit_studio_msgs/moveit_studio_vision_msgs/package.xml [hunk 1]
```diff
@@ -2,8 +2,8 @@
   <name>moveit_studio_vision_msgs</name>
   <version>3.0.0</version>
   <description>Messages, services, and actions used by MoveIt Pro vision capabilities.</description>
-  <author email="sebastian.castro@picknik.ai">Sebastian Castro</author>
-  <maintainer email="sebastian.castro@picknik.ai">Sebastian Castro</maintainer>
+  <author email="support@picknik.ai">MoveIt Pro maintainer</author>
+  <maintainer email="support@picknik.ai">MoveIt Pro maintainer</maintainer>
```
Comment (adlarkin) [inline moveit_studio_msgs/moveit_studio_vision_msgs/package.xml:6]: If you feel comfortable putting your name/email here, I think that's fine

**Review by adlarkin — APPROVED** [review]
I see this is a draft, but LGTM as-is. I can give another review if things change based on other related work.

**Comment by chancecardona** [comment]
@adlarkin thoughts on pointcloud vs point_cloud?

**Comment by adlarkin** [comment]
I think pointcloud (what you have) is fine!

**Review by EzraBrooks — APPROVED** [review]

**Pull request merged**